db/patch68.sql

```diff
@@ -3,4 +3,4 @@
 -- This resolves https://joindin.jira.com/browse/JOINDIN-725
 ALTER DATABASE joindin CHARACTER SET utf8 COLLATE utf8_unicode_ci;
 
-INSERT INTO patch_history SET patch_number = 68;
\ No newline at end of file
+INSERT INTO patch_history SET patch_number = 69;
```

Comment:
To further this, just checked and can see that patch_level 68 is not being set in the DB, likely as there are 2 patches declaring as 69.

https://www.dropbox.com/s/5ic7qwkza45s3h7/Screenshot%202017-02-02%2013.39.23.png?dl=0